sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content.py

```diff
@@ -383,3 +383,13 @@ def test_content_language_v2(self, client):
         with pytest.raises(ValueError) as e:
             client.begin_recognize_content(myfile, language="en")
         assert "'language' is only available for API version V2_1_PREVIEW and up" in str(e.value)
+
+    @FormRecognizerPreparer()
+    @GlobalClientPreparer()
+    def test_pages_kwarg_specified(self, client):
```

Comment:
I think we can skip this test for all content methods since we have a "real" test for pages called `test_content_specify_pages`

Reply:
Removed. Thanks for catching that!